kubefed/installing-kubefed.adoc

```diff
@@ -0,0 +1,21 @@
+[id="installing-kubefed"]
+= Installing OpenShift KubeFed
+include::modules/common-attributes.adoc[]
+include::modules/kubefed-document-attributes.adoc[]
+:context: installing-kubefed
+
+To get started with {KubeFedProductName}, you must create the required custom resources (CRs) and complete the Operator installation.
```

Comment:
The operator installation would be complete before you were able to create the CRs that drive installation.